scripts/measures/2019/enrich-measures-data.js

```diff
@@ -140,3 +143,41 @@ function addQualityStrataNames(measures) {
     });
   });
 }
+
+function addRequiredRegistrySubmissionMethod(measures) {
+  const eCQMeasures = measures.filter(m => m.eMeasureUuid !== undefined);
+  eCQMeasures.forEach(m => {
+    if (m.submissionMethods.includes('electronicHealthRecord') && !m.submissionMethods.includes('registry')) {
+      m.submissionMethods.push('registry');
+    }
+  });
+}
+
+/**
+ * Adds performance and eligibility options for claims-related measures.
+ *
+ * The source for this is a JSON file generated by /claims-related/scripts/single_source_to_json.py
+ * See /claims-related/README.md for more information.
+ */
+function enrichClaimsRelatedMeasures(measures) {
+  // these are the attributes we are interested in
+  const attributes = [
+    'eligibilityOptions',
+    'performanceOptions'
+  ];
+
+  // to avoid nested iteration, let's sort claims related measures by their measure ID
+  const claimsRelatedMeasures = JSON.parse(fs.readFileSync(path.join(__dirname, qppSingleSourceJson)));
+
+  // now for each measure, add the attributes from the claims-related measures set
+  measures.forEach(measure => {
+    // if the measure is in claimsRelatedMeasures, we need to merge its attributes
+    const claimsRelatedMeasure = claimsRelatedMeasures[measure.measureId];
+    if (claimsRelatedMeasure) {
+      for (const attribute of attributes) {
+        measure[attribute] = claimsRelatedMeasure[attribute];
+      }
+    }
+  });
+}
+
```

Comment:
This blank line is causing a linter failure. You can check for other linter failures after fixing it by running `npm run test` locally

Reply:
Thanks for the linter tip :+1:

Reply:
I updated the PR description that included the additional scope

Reply:
@sheldon-b 
Linter check passed
Rebased off master
Pr description updated
version bumped